src/daft-dsl/src/visitor.rs

```diff
@@ -299,18 +285,6 @@ impl<'py> PyVisitor<'py> {
         ))
     }
 
-    // Developer Note
-    // ==============
-    // These *could* be factored out as IntoPyObject implementations,
-    // but there are few caveats, and I wish to keep these conversions
-    // scoped to interactions with the visitor for now. For types like
-    // Expr and DataType we cannot implement the trait on the external
-    // type. We can implement IntoPyObject for LiteralValue, but again
-    // I wish to have these conversions limited in scope and usage.
-    // We don't want to implement IntoPyObject for PyExpr etc. since
-    // these types are already py objects, just different than the
-    // conversions we want here.
```

Comment:
Let's leave the other notes since they are important as to why the code is factored this way. I was just hoping to remove the one sentence which isn't relevant anymore.

> We can implement IntoPyObject for LiteralValue, but again I wish to have these conversions limited in scope and usage.